src/Input/RadioButton/RadioButton.test.tsx

```diff
@@ -102,4 +104,105 @@ describe('<RadioButton>', () => {
     expect(radioButton1).not.toBeChecked();
     expect(radioButton2).toBeChecked();
   });
+
+  it('should not call onChange again when disabled', () => {
+    const onChange = jest.fn();
+    const { getByTestId } = render(
+      <RadioButton
+        label="Full Time"
+        name="job-type"
+        value="full-time"
+        data-testid="radio"
+        onChange={onChange}
+        disabled={true}
+      />
+    );
+    const radioButton = getByTestId('radio');
+    userEvent.click(radioButton);
+    expect(onChange).not.toHaveBeenCalled();
+  });
+});
+
+describe('<RadioButton /> color', () => {
```

Comment:
Why not test this with snapshots?

Reply:
Checking explicit style/ variable is safer for me because I cannot ensure every developer will check the update to snapshots carefully. 🤔